sass/site/site.scss

```diff
@@ -158,7 +158,7 @@ $paddingNormal: 20px;
 // --------------------------------------------------
 .navbar {
   margin-bottom: 0px;
-  border-top: 3px solid $blueDark;
+  border-top: 3px solid #0d94d2; //cerner blue
```

Comment:
I am doubtful about it since I could only find these colors https://github.com/sujaysudheenda/cerner.github.io/blob/stylize-section-header/sass/bootstrap/_variables.scss#L33